woocommerce/woocommerce-gateway-stripe
# Remove `_wcstripe_feature_upe` flag and all traces of old settings

Fixes #2150

## Changes proposed in this Pull Request:

Here we want to remove deprecated code that comes after the introduction of the new settings UI on Stripe. Here's a list of the changes on this PR.

- Removes the usage `_wcstripe_feature_upe_settings`
- Removes the usage of `WC_Stripe_Feature_Flags::is_upe_settings_redesign_enabled()`
- Removes all traces of the old settings UI. Including SASS, CSS and JS files.
- Removes `WC_Stripe_UPE_Compatibility_Note` class. 







## Testing instructions



You'll need WooCommerce extension up and running. Finish KYC onboarding.

- Install WooCommerce Stripe extension
- Delete the `_wcstripe_feature_upe_settings` option. In case you already have it. 
  - Use `wp option delete _wcstripe_feature_upe_settings` or 
  - use `npm run wp option delete _wcstripe_feature_upe_settings` if Stripe running locally
- Go to "WooCommerce > Settings > Payments".
- Click "Set Up" in the Stripe row.
- You should see the new UI despite `_wcstripe_feature_upe_settings` has been deleted.
- <img width="587" alt="Screen Shot 2021-11-19 at 2 05 25 PM" src="https://user-images.githubusercontent.com/1025173/142677555-e5c73e0f-c972-412f-8b64-6cfb97acec5e.png">
- Click "Enter account keys" then click the "Test" tab and enter the test keys.
- You should see the new Settings UI.
- <img width="994" alt="Screen Shot 2021-11-19 at 2 15 34 PM" src="https://user-images.githubusercontent.com/1025173/142678820-fad601c8-1bb8-4fa8-b55e-c801e950e516.png">
- <img width="1056" alt="Screen Shot 2021-11-19 at 2 16 07 PM" src="https://user-images.githubusercontent.com/1025173/142678893-60942760-dc4a-429c-9435-d1481be730f2.png">
- Click "Customize appearance" in the "Payment Methods" tab.
- You should see the "Customize express checkouts" view
- <img width="981" alt="Screen Shot 2021-11-19 at 2 20 33 PM" src="https://user-images.githubusercontent.com/1025173/142679459-819d8b0e-ac94-4c2d-8c56-410344fb7ee7.png">

**The idea is to test the Stripe extension and confirm it works normally without any trace of the old design.**



---


**Post merge**
